modules/dev_overrides/docker-compose.world_modeling.yaml

```diff
@@ -13,7 +13,7 @@ services:
     image: "${WORLD_MODELING_HD_MAP_IMAGE}:build_${TAG}"
     command: tail -F anything
     volumes:
-      - ${MONO_DIR}/src/world_modeling:/home/ament_ws/src/hd_map
+      - ${MONO_DIR}/src/world_modeling:/home/bolty/ament_ws/src/hd_map
```

Comment:
oh @VishGit1234 can this be a separate PR to fix all the volume directory mismatches?

Reply:
done